src/palace/manager/api/bibliotheca.py

```diff
@@ -234,60 +234,43 @@ def signature(self, method, path):
         signature = base64.standard_b64encode(digest)
         return signature, now
 
-    def full_url(self, path):
+    def full_url(self, path: str) -> str:
         if not path.startswith("/cirrus"):
             path = self.full_path(path)
         return urllib.parse.urljoin(self.base_url, path)
 
-    def full_path(self, path):
+    def full_path(self, path: str) -> str:
         if not path.startswith("/"):
             path = "/" + path
         if not path.startswith("/cirrus"):
             path = f"/cirrus/library/{self.library_id}{path}"
         return path
 
-    def request(self, path, body=None, method="GET", identifier=None, max_age=None):
+    def request(
+        self,
+        path: str,
+        body: str | None = None,
+        method: str = "GET",
+    ) -> Response:
         path = self.full_path(path)
         url = self.full_url(path)
         if method == "GET":
             headers = {"Accept": "application/xml"}
         else:
             headers = {"Content-Type": "application/xml"}
         self.sign(method, headers, path)
-        # print headers
```

Comment:
I dropped the `max_age` parameter and the logic to handle it here, because it wasn't used, and because it makes the response type of this function complicated since `Representation.get` has a different return type from `self._request_with_timeout`. This seemed needlessly complicated, when it wasn't actually used.